### ISSUE TITLE
not-translated:What is \lvert -8.5 \rvert∣−8.5∣open ...

### ISSUE THREAD

**Issue by KhanBugz** [Khan/khan-i18n#118732]
Url: https://pt.khanacademy.org/mission/algebra-basics

User Agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.87 Safari/537.36

Problem Text: 
What is \lvert -8.5 \rvert∣−8.5∣open vertical bar, minus, 8, point, 5, close vertical bar?

Hint #1

\small{-10}−10\small{-9}−9\small{-8}−8\small{-7}−7\small{-6}−6\small{-5}−5\small{-4}−4\small{-3}−3\small{-2}−2\small{-1}−1\small{0}0
Hint #2
The distance from 000 to -8.5−8.5minus, 8, point, 5 is 8.58.58, point, 5, which equals the absolute value.
Hint #3
In other words, 8.58.58, point, 5 is the non-negative version of -8.5−8.5minus, 8, point, 5.

User Description: 

Next Steps: 
If this is a translation issue, fix it. Otherwise verify what is wrong, add a comment in English describing the issue and assign to tzjames.
